dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -700,6 +700,10 @@ error.package.extension=CWWKE0950E: The archive filename must end with .jar when
 error.package.extension.explanation=The --archive option must include a file extension that ends with .jar when specifying --include=runnable.
 error.package.extension.useraction=Supply a file name to --archive with a file extension of .jar when specifying --include=runnable.
 
+error.package.usr.jar=CWWKE0951E: The archive filename must not end with a .jar file extension when the usr option is used.
```

Comment:
Change
"The archive **filename** must not end with a .jar file extension when the usr option is used."
to
"The archive **file name** must not end with a .jar file extension when the usr option is used."

Reply:
Hey @Charlotte-Holt 

I have fixed all 3 of these and pushed them up.  Thank you for the review!